velox/expression/tests/SparkExpressionFuzzerTest.cpp

```diff
@@ -47,6 +47,7 @@ int main(int argc, char** argv) {
   // rlike, md5 and upper
   std::unordered_set<std::string> skipFunctions = {
       "regexp_extract",
+      "regexp_replace",
```

Comment:
nit: would you add a comment with a link to GitHub issue where we have discussed why we have to skip this function?